providers/implementations/include/prov/ml_kem.h

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2024 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#ifndef OSSL_INTERNAL_ML_KEM_H
+# define OSSL_INTERNAL_ML_KEM_H
+# pragma once
+
+# ifndef OPENSSL_NO_ML_KEM
+
+#  include <stdint.h>
```

Comment:
This header appeared with C99...  which makes this inclusion problematic with older compilers (which are still in use!).

We have definitions in `<openssl/e_os2.h>` that compensate...  but you're already including that in `<crypto/ml_kem.h>`, so that begs the question, what is missing that prompted you to include `<stdint.h>`?

Reply:
Please remove it, then...

Reply:
Sure. Coming right up...

Reply:
Actually, looks like I've already done it. :-)